.travis.yml

```diff
@@ -19,7 +25,10 @@ env:
     - BROWSER: chrome
     - BROWSER: firefox
     - BROWSER: safari
-    - BROWSER: ie9..11
-    - BROWSER: ipad6..8
-    - BROWSER: iphone6..8
+    - BROWSER: ie:11
+    - BROWSER: ie:10
+    - BROWSER: ie:9
+    - BROWSER: iphone:8
+    - BROWSER: iphone:7
+    - BROWSER: iphone:6
```

Comment:
I dropped ipad 6-8 as iphone should cover our testing needs. Open to hearing arguments against.